README.md

```diff
@@ -1,6 +1,15 @@
 ## Butterfree
 _A tool for building feature stores. Transform your raw data into beautiful features._
 
+
+[![Release](https://img.shields.io/github/v/release/quintoandar/butterfree)]((https://pypi.org/project/butterfree/))
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
```

Comment:
another cool badge to add could be:
- [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
- [![Pypi](https://img.shields.io/pypi/v/butterfree)](https://pypi.org/project/butterfree/)
- [![Downloads](https://pepy.tech/badge/butterfree)](https://pepy.tech/project/butterfree) (you could get other monthly/weekly counts too here: https://pepy.tech/project/butterfree)